src/react-image-lightbox.js

```diff
@@ -1180,6 +1191,15 @@ class ReactImageLightbox extends Component {
     this.getSrcTypes().forEach(srcType => {
       const type = srcType.name;
 
+      // there is no error when we try to load it initially
+      if (props[type]) {
+        this.setState(prevState => ({
+          loadErrorStatus: Object.assign({}, prevState.loadErrorStatus, {
```

Comment:
Please use the object rest spread operator, like so:
```js
{ ...prevState.loadErrorStatus, [type]: false }
```
rather than `Object.assign`. This is partially about code style, partially about browser compatibility (Babel transpiles the spread operator code to ES5)

Reply:
sounds good!